website/docs/d/backup_compliance_policy.html.markdown

```diff
@@ -0,0 +1,178 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: backup_compliance_policy"
+sidebar_current: "docs-mongodbatlas-datasource-backup-compliance-policy"
+description: |-
+    Provides a Backup Compliance Policy Datasource.
+---
+
+# Data Source: mongodbatlas_backup_compliance_policy
+
+`mongodbatlas_backup_compliance_policy` provides a Backup Compliance Policy data source. An Atlas Backup Compliance Policy provides the current protection policy settings for the project. Prevent any user, regardless of role, from modifying or deleting specific cluster configurations and backups. When enabled, the Backup Compliance Policy will be applied as the minimum policy for all clusters and can only be disabled by contacting MongoDB support. Only supported for clusters M10 or higher.
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find `groupId` in the official documentation.
+
+## Example Usage
+
+```terraform
+resource "mongodbatlas_cluster" "my_cluster" {
+  project_id   = "<PROJECT-ID>"
+  name         = "clusterTest"
+
+  //Provider Settings "block"
+  provider_name               = "AWS"
+  provider_region_name        = "EU_CENTRAL_1"
+  provider_instance_size_name = "M10"
+  provider_backup_enabled     = true // enable cloud backup snapshots
+}
+
+resource "mongodbatlas_cloud_provider_snapshot_backup_policy" "test" {
```

Comment:
@martinstibbe this is deprecated resource, can you update like how we did in resource example?